### DISCUSSION TITLE
Маппинг данных ФРМО на FHIR ресурсы

### DISCUSSION THREAD

**Discussion by iradio** [fhir-ru/core#26] [Административная РГ]
Состав сведений, планируемых к сбору через новую версию подсистем ФРМО ЕГИСЗ в 2022 году, представлен здесь
https://docs.google.com/spreadsheets/d/1GGGzZTmJ7yVrEUCTu9AO8Q1epMoaZ08C/edit#gid=2043079596 (все могут комментировать)
В столбце "Поле доступно для организаций" кодируются типы организаций по следующему словарю:

- ЛО - лицензируемая организация , те новые организации которые собирают данные для получения лицензии (ещё не МО, ФО и ОО)
- МО - Мед.Организация 
- ФО - Фармацевтическая  Организация
- ОО- образовательная организация , они уже сейчас есть в фрмо
- ОЗ - Организатор Здравоохранения

Вопрос к сообществу - как выстроить маппинг данных ФРМО на структуру ресурсов FHIR RUcore.
Базовое предположение:

- Общие данные юридического лица - Organzation https://www.hl7.org/fhir/organization.html
- Адресные и геоданные (кроме адреса юр.лица) - Location https://www.hl7.org/fhir/location.html
- Специфика МО, ФО, ОО и ОЗ - HealthcareService https://www.hl7.org/fhir/healthcareservice.html 
- Структурные подразделения - ?
- Территориально-выделенные структурные подразделения - ?

Но тогда нужно будет сделать несколько профилей под разные HealthcareService, т.к. атрибутивный состав МО, ФО, ОО и ОЗ заметно различается на уровне обязательных полей.

Полезное: описание Административного модуля FHIR для понимания взаимосвязей https://www.hl7.org/fhir/administration-module.html

**Comment by OlegPenzin** [comment]
Нужно не забыть про структурные подразделения (СП) - под них, вероятно, придётся выделить отдельный профиль Organization, позволяющий собирать иерархию оргструктуры

**Reply by iradio** [reply]
Расширю - тут нужно не забыть не только про Структурные подразделения (СП), но и про Территориально выделенные структурные подразделения (ТВСП). Множество показателей Минздрава по нац.проектам построены на подсчете именно ТВСП. Определение ТВСП дано в паспорте фед.проекта https://minzdrav.gov.ru/poleznye-resursy/natsproektzdravoohranenie/tsifra

Территориально-выделенное структурное подразделение медицинской организации (ТВСП МО) - здание или совокупность зданий, расположенных по отдельному адресу, в котором (-ых) располагаются структурные подразделения медицинской организации, оказывающие первичную, специализированную и прочие виды медицинской помощи.

В структуре данных ФРМО https://docs.google.com/spreadsheets/d/1GGGzZTmJ7yVrEUCTu9AO8Q1epMoaZ08C/edit#gid=2043079596 они размечены соответствующими разделами:

- Сведения о структурных подразделениях (далее – СП) медицинской организации
- Сведения о территориально выделенных структурных подразделениях медицинской организации

**Reply by iradio** [reply]
Дополнил постановку.

**Comment by DmFadin** [comment]
А каким образом планируется использовать эти данные и как будет выглядеть организация со сложной структурой? например больничная аптека в составе МО?

**Comment by Vrodvi** [comment]
ПРЕДЛОЖЕНИЕ №1:
Предлагаю до начала какого-то маппинга определить основные термины и определения с которыми предстоит работать.
А также составить "логическую" структуру информации об организациях и сопутствующих понятиях (филиалы, подразделения, лицензии и прочее).

Что именно подразумевается под "организацией", например?
Имеется ввиду юридическое лицо, зарегистрированное в ОГРН?
Или имеется ввиду один из адресов юридического лица или наименование конкретной территории или строения, где непосредственно оказываются медицинские (и сопутствующие) услуги?

Нужно уточнение по указанной классификации:
> ЛО - лицензируемая организация , те новые организации которые собирают данные для получения лицензии (ещё не МО, ФО и ОО)
> МО - Мед.Организация
> ФО - Фармацевтическая Организация
> ОО- образовательная организация , они уже сейчас есть в фрмо
> ОЗ - Организатор Здравоохранения
Одно юридическое лицо может иметь лицензии на осуществление нескольких типов деятельности - например, иметь лицензии как на осуществление медицинской, так и образовательной деятельности.
Означает ли это, что при осуществлении нескольких типов деятельности появляется несколько "организаций"?

Достаточно ли указания адреса, по которым "организация" осуществляет свою деятельность оформить в виде 
> "Адресные и геоданные (кроме адреса юр.лица) - Location https://www.hl7.org/fhir/location.html"
Ведь врач работает не в юридическом лице и не просто по адресу, врач имеет должность в одном из структурных подразделений и\или отделений.

**Reply by niquola** [reply]
Немного погрузился в тему,  чтобы гармонизироваться с юредическими определениями имеет смысл за основу взять Юря Лицо для описания организации (тянет на самостоятельный базовый профайл) и Структурные Подразделения  для описания структуры

**Comment by Vrodvi** [comment]
ПРЕДЛОЖЕНИЕ №2:
Предлагаю взять одну или несколько конкретных медицинских организаций со сложной организационной структурой (а также пример сотрудника этой организации) и на этих примерах расписать все составляющие элементы.

Один из примеров:

**"ГБУЗ МКНЦ имени А.С. Логинова ДЗМ", ОГРН 1027700189626** 

Это юридическое лицо имеет множество адресов, по которым осуществляется медицинская деятельность в соответствии с лицензией  - см. https://mknc.ru/contents/posts/0/1/licenziya-gbuz-mknc-imeni-a-s-loginova-dzm-24-12-2019.pdf:
109451, г. Москва, б-р Перервинский, д. 5, к. 1
111123, г. Москва, ул. Новогиреевская, д. 1
111123, г. Москва, ул. Новогиреевская, д. 1, корп. 2
111123, г. Москва, ул. Новогиреевская, д. 1, корп. 3
111123, г. Москва, ул. Новогиреевская, д. 1, корп. 4
111123, г. Москва, ш. Энтузиастов, д. 86, стр. 12
111123, г. Москва, шоссе Энтузиастов, д. 86
111123, г. Москва, шоссе Энтузиастов, д. 86, стр. 2
111123, г. Москва, шоссе Энтузиастов, д. 86, стр. 6
115172, г. Москва, Гончарная улица, д. 23, стр. 1
115172, г. Москва, Гончарная улица, д. 23, стр. 2
121552, г. Москва, ул. Академика Павлова, д. 22
123242, г. Москва, Верхний Предтеченский переулок, д. 8, стр. 1
123242, г. Москва, Верхний Предтеченский переулок, д. 8, стр. 2 

Также данное юридическое лицо имеет множество адресов, по которым осуществляется образовательная деятельность в соответствии с лицензией - см. https://mknc.ru/contents/posts/2627000/2627649/document3486647.pdf:
111123, г. Москва, шоссе Энтузиастов, дом 86, строение 6
место нахождения юридического лица или его филиала
111123, г. Москва, ул. Новогиреевская, д. 1;
111123, г. Москва, ул. Новогиреевская, д. 1, корп. 2;
111123, г. Москва, ул. Новогиреевская, д. 1, корп. 3;
111123, г. Москва, ул. Новогиреевская, д. 1, корп. 4;
111123, г. Москва, шоссе Энтузиастов, д. 86;
111123, г. Москва, шоссе Энтузиастов, д. 86, стр. 12

При этом логически в МКНЦ выделено несколько "филиалов" (некоторые из них (КЖЗ) также имеют несколько филиалов):

1) ГБУЗ Московский Клинический Научный Центр имени А. С. Логинова ДЗМ
	111123, г. Москва, шоссе Энтузиастов, 86

2) Филиал «Маммологический центр» (Клиника Женского Здоровья) ГБУЗ МКНЦ имени А.С. Логинова ДЗМ (далее МЦ «КЖЗ»)
	- Филиал №1 – г. Москва, ул. Гончарная д. 23
	- Филиал №2 – г. Москва, Верхний Предтеченский переулок д. 8
	- Филиал №3 – г. Москва, шоссе Энтузиастов д. 86 (7 корпус, 7 этаж)
	- Филиал №4 – г. Москва, ул. Академика Павлова, д. 22	

3) Центр амбулаторной онкологической помощи (ЦАОП) Филиал ГБУЗ города Москвы «Московский клинический научно-практический центр имени А.С. Логинова Департамента здравоохранения города Москвы»
	109451, г. Москва, Перервинский б-р, д.5, кор.1
	
4) Филиал МКНЦ имени А.С. Логинова на Павлова
	ул. Академика Павлова, 22

Каждый филиал, в свою очередь, имеет определённую организационную структуру из отделений\подразделений\кабинетов\и т.п.

Какова будет логическая структура обозначенных объектов?

- <<юридическое лицо>> ГБУЗ МКНЦ имени А.С. Логинова ДЗМ", ОГРН 1027700189626
	- <<медицинская организация???>> ГБУЗ МКНЦ имени А.С. Логинова ДЗМ"
		- <<основное структурное подразделение???>> ГБУЗ Московский Клинический Научный Центр имени А. С. Логинова ДЗМ
		- <<филиал>> Филиал «Маммологический центр» (Клиника Женского Здоровья) ГБУЗ МКНЦ имени А.С. Логинова ДЗМ (далее МЦ «КЖЗ»)
			- <<филиал филиала>> Филиал №1
				- <<структурное подразделение: отделение???>> Консультативно-диагностическое отделение
					- <<структурное подразделение: кабинет???>> Кабинет №5
			- <<филиал филиала>> Филиал №2
			- <<филиал филиала>> Филиал №3
			- <<филиал филиала>> Филиал №4
		- <<филиал>> Центр амбулаторной онкологической помощи (ЦАОП) Филиал ГБУЗ города Москвы «Московский клинический научно-практический центр имени А.С. Логинова Департамента здравоохранения города Москвы»
		- <<филиал>> Филиал МКНЦ имени А.С. Логинова на Павлова
	- <<образовательная организация???>> ГБУЗ МКНЦ имени А.С. Логинова ДЗМ"

Предположим, что врач-онколог Иванов Иван Иванов осуществляет приём в кабинете №5 консультативно-диагностического отделения в Филиале №1 (г. Москва, ул. Гончарная д. 23) «Маммологического центра» ГБУЗ МКНЦ имени А.С. Логинова ДЗМ.
Как следует обозначать данную занятость врача?

**Reply by niquola** [reply]
Филиалы являются типом структурных подразделений (http://www.consultant.ru/document/cons_doc_LAW_17819/2dde5484151bb43287266805650ce09ee9b973ba/)

Вроде как не может быть ["филиала филиалов" ](https://www.law.ru/question/6121-mojet-li-filial-kompanii-sozdavat-filial-ili-inoe-obosoblennoe-podrazdelenie-chtoby-eto#:~:text=%3A%20%D0%9E%D0%B1%D0%BE%D1%81%D0%BE%D0%B1%D0%BB%D0%B5%D0%BD%D0%BD%D1%8B%D0%B5%20%D0%BF%D0%BE%D0%B4%D1%80%D0%B0%D0%B7%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F%20%D0%BC%D0%BE%D0%B3%D1%83%D1%82%20%D1%81%D1%83%D1%89%D0%B5%D1%81%D1%82%D0%B2%D0%BE%D0%B2%D0%B0%D1%82%D1%8C%20%D1%82%D0%BE%D0%BB%D1%8C%D0%BA%D0%BE,%D0%BB%D0%B8%D1%86%D0%B0%2C%20%D0%BA%D0%B0%D0%BA%D0%BE%D0%B2%D1%8B%D0%BC%20%D1%84%D0%B8%D0%BB%D0%B8%D0%B0%D0%BB%20%D0%BD%D0%B5%20%D1%8F%D0%B2%D0%BB%D1%8F%D0%B5%D1%82%D1%81%D1%8F.&text=%D0%98%D0%BC%D0%B5%D1%82%D1%8C%20%D0%B2%20%D1%81%D0%B2%D0%BE%D0%B5%D0%BC%20%D1%81%D0%BE%D1%81%D1%82%D0%B0%D0%B2%D0%B5%20%D0%B4%D1%80%D1%83%D0%B3%D0%B8%D0%B5,55%20%D0%93%D0%9A%20%D0%A0%D0%A4)

**Reply by RodVit** [reply]
Юридически не может, но управленчески, видимо, может. Я ведь привел реальный пример -  МКНЦ имени А.С. Логинова, у его филиала КЖЗ указаны ещё 4 филиала по разным адресам. 
Если это не юридическая конструкция, то нужно понять что это и как мы это деление в ресурсах отразим.

**Comment by Vrodvi** [comment]
Ещё один пример для разбора:

**ФГБУ «НМИЦ радиологии» Минздрава России, ОГРН 1027739623031** 	
	
Это юридическое лицо имеет множество адресов, по которым осуществляется медицинская деятельность в соответствии с лицензией - см. https://nmicr.ru/upload/doc/2017/Licenz_na_meddey_NFS-99-01-009432_ot_%2003_10_2017.pdf, https://nmicr.ru/about/dokumenty/doc/%D0%BC%D0%B5%D0%B4.%D0%B4%D0%B5%D1%8F%D1%82%D0%B5%D0%BB%D1%8C%D0%BD%D0%BE%D1%81%D1%82%D1%8C%20%D0%A4%D0%A1-40-01-000769%20%D0%BE%D1%82%2025.06.2020.pdf:	
125284, г. Москва, 2-ой Боткинский проезд, д. 3, стр. 1
125284, г. Москва, 2-ой Боткинский проезд, д. 3, стр. 2
125284, г. Москва, 2-ой Боткинский проезд, д. 3, стр. 2а
125284, г. Москва, 2-ой Боткинский проезд, д. 3, стр. 3
125284, г. Москва, 2-ой Боткинский проезд, д. 3, стр. 10
119121, г. Москва, ул. Погодинская, д. 6, стр. 1
105425, г. Москва, ул. 3-я Парковая, д. 42А
105425, г. Москва, ул. 3-я Парковая, д. 51, стр. 1
105425, г. Москва, ул. 3-я Парковая, д. 51, стр. 2
105425, г. Москва, ул. 3-я Парковая, д. 51, стр. 4
249036, Калужская область, г. Онинск, ул. Королева, д. 4
249036, Калужская область, г. Онинск, ул. Королева, д. 2, здание 15
249032, Калужская область, г. Обнинск, Киевское шоссе, здание 3А, строение 2
249032, Калужская область, г. Обнинск, Киевское шоссе, здание 3А, строение 3

Также данное юридическое лицо имеет множество адресов, по которым осуществляется образовательная (https://nmicr.ru/about/dokumenty/doc/licenziya-na-osushhestvlenie-obrazovatel%60noj-deyatel%60nosti-ot-03-noyabrya-2017g.pdf) и фармацевтическая (https://nmicr.ru/upload/doc/2017/licenz_na_farm_deyt_N_FS_77_02_001109_ot_18_09_2017.pdf и https://nmicr.ru/upload/doc/2017/MNRC_%2020170905_licenz_na_farm_deyt_FC_%2040-02-000095_ot_5_09_2017.pdf) деятельность в соответствии с лицензиями.

При этом логически в НМИЦ радиологии выделено несколько "филиалов":

- ФГБУ «НМИЦ радиологии» Минздрава России
125284, г. Москва, 2-ой Боткинский проезд, д. 3
	- МНИОИ им. П.А.Герцена - филиал ФГБУ «НМИЦ радиологии» Минздрава России 
	125284, г. Москва, 2-ой Боткинский проезд, д. 3
	- НИИ урологии и интервенционной радиологии им. Н.А. Лопаткина - филиал ФГБУ «НМИЦ радиологии» Минздрава России
	105425, г. Москва, ул. 3-я Парковая, д. 51
	- МРНЦ им. А.Ф. Цыба - филиал ФГБУ «НМИЦ радиологии» Минздрава России 
	249036, Калужская область, г. Онинск, ул. Королева, д. 4

Какова будет логическая структура обозначенных объектов?

- <<юридическое лицо>> ФГБУ «НМИЦ радиологии» Минздрава России, ОГРН 1027739623031
	- <<медицинская организация???>> ФГБУ «НМИЦ радиологии» Минздрава России
		- <<филиал>> МНИОИ им. П.А.Герцена - филиал ФГБУ «НМИЦ радиологии» Минздрава России
		- <<филиал>> НИИ урологии и интервенционной радиологии им. Н.А. Лопаткина - филиал ФГБУ «НМИЦ радиологии» Минздрава России
		- <<филиал>> МРНЦ им. А.Ф. Цыба - филиал ФГБУ «НМИЦ радиологии» Минздрава России 
	- <<образовательная организация???>> ФГБУ «НМИЦ радиологии» Минздрава России
	- <<фармацевтическая организация???>> ФГБУ «НМИЦ радиологии» Минздрава России

**Comment by eugenekogan-spb** [comment]
Коллеги, я попытался сопоставить структуру ФРМО и структуру ресурсов FHIR, описывающих те же вещи.
Получилось следующее:
<img width="979" alt="image" src="https://user-images.githubusercontent.com/62525329/143614978-624e2b15-5ad3-47e1-8cc4-c78c86c68849.png">
FHIR предлагает определить точки оказания услуг, это иерархическая структура, каждый узел которой может быть своего типа. Кроме того, он предлагает определить структуру организации (подчиненности) тоже до необходимого уровня. Я на картинку добавил (штатное) расписание и врачей, чтобы видеть, что на что ссылается. Картинка специально не полная, скажем, не показаны ресурсы, чтобы сначала договориться о базовых вещах.

В то же время структура ФРМО, как мне показалось, может быть отображена следующей картинкой:

<img width="899" alt="image" src="https://user-images.githubusercontent.com/62525329/143615656-e673d426-5e2d-46de-bbee-91b80389a89b.png">

Если все это верно (прошу замечания), то для начала похоже надо разложить информацию ФРМО на то, что относится к оргструктуре и то, что относится к точкам оказания услуг.

**Reply by RodVit** [reply]
Где на этих схемах юридическое лицо, филиалы?

Чем ТВСП отличается от просто структурного подразделения? Территориально выделено относительно чего? 

Почему кабинет - не структурное подразделение с типом "кабинет"?  

Что на схеме делает здание, это ведь просто часть адреса?

Где на схеме МО, ОО, ФО? Как будет изменена схема когда одно юридическое лицо имеет лицензии на все эти виды деятельности? Это будут отдельные организации? Что тогда такое организация, одна лицензия - одна организация?


На схеме также отсутствует отношение врача к структуре организации - к чему предполагается относить занятость врача?

Про структуру в fhir всё понятно - почему выделены конкретно такие ресурсы описано в каждом из этих ресурсов с пояснениями.
Где в ФРМО найти объяснения по принятой структуре, на основании чего принимались решения по этой структуре?

**Reply by AndreyVEgorov** [reply]
> 

![DHEP](https://user-images.githubusercontent.com/87869738/144094346-ef3659ac-ccb5-4860-9fd7-94080c042f35.png)
Я в свое время реализовал такую структуру

**Comment by DmFadin** [comment]
Мне кажется стоит разделить 4 сущности: Организация обладатель лицензии и ее юридическая структура, Функциональная структура организации, Места (адреса) оказания помощи (с учетом возможности выезда), Медицинские работники которые являются сотрудниками организации. Разница между структурами данных возникает оттого, что носителем лицензии является в РФ организация, а не врач. Кроме того хорошо бы договориться, зачем будет использоваться каждый ресурс, и если мы не можем найти ему применение, то предлагаю в структуру данных его не заводить

**Comment by niquola** [comment]
# Организационная Структура

описыватеся базовым профайлом для Юр Лица, от которого наследуется Мед Организация
Структура МедОрганизации предствалена в виде Структурных Подразделений  (которые могут быть обособленными - филиалы, представительства, и не обособленными - http://www.consultant.ru/document/cons_doc_LAW_162754/4da093376d8eaf873ed27765b61cec8a72e13bd9/)
Здания и помещения описываются Location связанным либо с подразделением либо с Мед Организацией

![image](https://user-images.githubusercontent.com/32066/144726704-4fad4772-467d-4e72-8673-3112fcb17352.png)


## Организация

****ru:Organization**

Организация  является [Юредическим Лицом](http://www.consultant.ru/document/cons_doc_LAW_5142/68642eb1daeec13480d8f283f27bc14b42b929df/) и типизацией для нашего домена, позволяющей построить дочерние профайлы

Обязательные поля:

- Наименование организации;
-  виды юр лица - https://ru.m.wikipedia.org/wiki/%D0%AE%D1%80%D0%B8%D0%B4%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D0%BB%D0%B8%D1%86%D0%BE
- тип организации:
  * ЛО - лицензируемая организация , те новые организации которые собирают данные для получения лицензии (ещё не МО, ФО и ОО)
  * МО - Мед.Организация
  * ФО - Фармацевтическая Организация
  * ОО- образовательная организация , они уже сейчас есть в фрмо
  * ОЗ - Организатор Здравоохранения
-  Юридический адрес (В случае ИП – адрес места регистрации Предпринимателя);
-  Идентификаторы
   • [ОГРН](https://ru.wikipedia.org/wiki/%D0%9E%D1%81%D0%BD%D0%BE%D0%B2%D0%BD%D0%BE%D0%B9_%D0%B3%D0%BE%D1%81%D1%83%D0%B4%D0%B0%D1%80%D1%81%D1%82%D0%B2%D0%B5%D0%BD%D0%BD%D1%8B%D0%B9_%D1%80%D0%B5%D0%B3%D0%B8%D1%81%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%BE%D0%BD%D0%BD%D1%8B%D0%B9_%D0%BD%D0%BE%D0%BC%D0%B5%D1%80) - Основной государственный регистрационный номер
   • [ИНН](https://ru.wikipedia.org/wiki/%D0%98%D0%B4%D0%B5%D0%BD%D1%82%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%86%D0%B8%D0%BE%D0%BD%D0%BD%D1%8B%D0%B9_%D0%BD%D0%BE%D0%BC%D0%B5%D1%80_%D0%BD%D0%B0%D0%BB%D0%BE%D0%B3%D0%BE%D0%BF%D0%BB%D0%B0%D1%82%D0%B5%D0%BB%D1%8C%D1%89%D0%B8%D0%BA%D0%B0) - Идентификационный номер налогоплательщика
   • [КПП](https://ru.wikipedia.org/wiki/%D0%98%D0%B4%D0%B5%D0%BD%D1%82%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%86%D0%B8%D0%BE%D0%BD%D0%BD%D1%8B%D0%B9_%D0%BD%D0%BE%D0%BC%D0%B5%D1%80_%D0%BD%D0%B0%D0%BB%D0%BE%D0%B3%D0%BE%D0%BF%D0%BB%D0%B0%D1%82%D0%B5%D0%BB%D1%8C%D1%89%D0%B8%D0%BA%D0%B0#%D0%9A%D0%BE%D0%B4_%D0%BF%D1%80%D0%B8%D1%87%D0%B8%D0%BD%D1%8B_%D0%BF%D0%BE%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%BA%D0%B8_%D0%BD%D0%B0_%D1%83%D1%87%D1%91%D1%82_(%D0%9A%D0%9F%D0%9F)) - Код причины постановки на учёт (КПП)

   • [ОКПО](https://ru.wikipedia.org/wiki/%D0%9E%D0%B1%D1%89%D0%B5%D1%80%D0%BE%D1%81%D1%81%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9_%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80_%D0%BF%D1%80%D0%B5%D0%B4%D0%BF%D1%80%D0%B8%D1%8F%D1%82%D0%B8%D0%B9_%D0%B8_%D0%BE%D1%80%D0%B3%D0%B0%D0%BD%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D0%B9) - Общероссийский классификатор предприятий и организаций
   • [ОКВЭД](https://ru.wikipedia.org/wiki/%D0%9E%D0%B1%D1%89%D0%B5%D1%80%D0%BE%D1%81%D1%81%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9_%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80_%D0%B2%D0%B8%D0%B4%D0%BE%D0%B2_%D1%8D%D0%BA%D0%BE%D0%BD%D0%BE%D0%BC%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B9_%D0%B4%D0%B5%D1%8F%D1%82%D0%B5%D0%BB%D1%8C%D0%BD%D0%BE%D1%81%D1%82%D0%B8) - Общероссийский классификатор видов экономической деятельности


FHIR Account

При заключении договора также используются реквизиты для оплаты – банковские реквизиты организации
   • Расчетный счет организации
   • Корреспондентский счет банка
   • БИК банка

### Структура организации

Организация может состоять из **Структурных Подразделений**, которые не являются самостоятельными юридичискими
лицами. Обособленные СП должны регистрироваться в Налоговой Службе.


## Медицинская Организация (MO)

**ru:MedOrganization** < ru:Organization (type = MO)

[Источник](http://www.consultant.ru/document/cons_doc_LAW_121895/b819c620a8c698de35861ad4c9d9696ee0c3ee7a/)

медицинская организация - Юридическое лицо независимо от организационно-правовой формы, 
осуществляющее в качестве основного (уставного) вида деятельности медицинскую деятельность на основании лицензии, 
предоставленной в порядке, установленном [законодательством](http://www.consultant.ru/document/cons_doc_LAW_113658/6a4a5b5468ba8b99831699f7d048d2a5d7710610/#dst100146)
Российской Федерации о лицензировании отдельных видов деятельности.
Положения настоящего Федерального закона, регулирующие деятельность медицинских организаций, 
распространяются на иные юридические лица независимо от организационно-правовой формы, 
осуществляющие наряду с основной (уставной) деятельностью медицинскую деятельность,
и применяются к таким организациям в части, касающейся медицинской деятельности.
В целях настоящего Федерального закона к медицинским организациям приравниваются индивидуальные предприниматели,
осуществляющие медицинскую деятельность;




* Тип: ФРМО. Номенклатура медицинских организаций по виду медицинской деятельности (https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.99.2.289);
* Идентификаторы
  *  OID [ФРМO](https://support.egisz.rosminzdrav.ru/index.php?/Knowledge/List/Index/70) - Федеральный реестр медицинских организаций (ФРМО)
    Реестр МО РФ (https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.11.1461**;


## Структурное Подразделение МО

**ru:MedDepartment** - Organization (type = СП)

Структура организации может быть представленна как иерархия **Структурных Подразделений**

В этом документе детальное описание https://mednet.ru/images/stories/files/2018-_OID_12-1-.pdf


* название
* тип:
  * Юр. Тип - обособленное (филиал, представительство)
  * уровни по ФРМО (1 - обособленные, 2 - структурные, 3 - первичные)
  * ФРМО. Типы структурных подразделений, участвующих в оказании медицинской помощи (https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.99.2.299);
* идентификаторы:
  *  ФРМО OID -  Справочник структурных подразделений (https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.99.2.114);
* partOf (часть** - ссылка на МО или родительское СП


## Здания и Помещения МО

** ru:MedLocation** - Location

**Reply by niquola** [reply]
Вопросы: 

* Где описывать лицензию - хотя она в структурированном виде похоже не существует -  Номенклатура медицинских организаций по виду медицинской деятельности видимо частично покрывает этот вопрос
* Есть ли структуры объединяющие несколько Юр Лиц
* Я также видел тип организации как Гос Юр Лицо

**Reply by toropok** [reply]
КПП и оквэд конечно НЕ идентификаторы предприятия. Предприятие не обязано быть юридическим лицом.

**Comment by RodVit** [comment]
Лицензии определяют тип организации - МО, ФО, ОО.
В представленной схеме лицензии будут косвенно обозначены через
дублирование организации под юридическим лицом (для случая когда юр.оицо
имеет лицензии на несколько видов деятельности).

**Comment by Avazkh** [comment]
Предложение: дано, что Organization - это юридическое лицо (определение юр.лица - статья 48 ГК РФ). Есть еще две сущности: "обособленное структурное подразделение" и "структуроное подразделение". Может, имеет смысл отличать их таким образом: "обособленное..." - территориально или функционально обособленная часть организации, являющееся юридическим лицом, единственным участником которого является другое юридическое лицо (в нашем случае - "организация"). А "структурное подразделение" - территориально или функционально обособленная часть организации, НЕ являющееся юридическим лицом.

**Comment by uestas** [comment]
[Структура ФРМО.xlsx](https://github.com/fhir-ru/core/files/7662277/default.xlsx)
[Структура ФРМО сопровождение.pdf](https://github.com/fhir-ru/core/files/7662278/default.pdf)

**Reply by Vrodvi** [reply]
Ссылки не работают

**Reply by uestas** [reply]
Починил

**Reply by Avazkh** [reply]
> [Структура ФРМО.xlsx](https://github.com/fhir-ru/core/files/7662277/default.xlsx) [Структура ФРМО сопровождение.pdf](https://github.com/fhir-ru/core/files/7662278/default.pdf)

Очень подробная информация. Круто! Но мы же не хотим переносить всё это в ресурс organization, верно?

**Reply by eugenekogan-spb** [reply]
Я бы предложил не вносить все это в core Organization. Надо сделать профиль ФРМО и поля, не используемые при поддержке медицинских процессов, определить в нем.

**Reply by Vrodvi** [reply]
@uestas Коллеги из ЦНИИОИЗ, можно вас попросить заполнить предоставленную структуру ФРМО реальными данными по двум предложенным организациям:
ФГБУ «НМИЦ радиологии» Минздрава России, ОГРН 1027739623031
"ГБУЗ МКНЦ имени А.С. Логинова ДЗМ", ОГРН 1027700189626
Это позволит более чётко понять деление организации на составные части.

**Comment by shamilnn** [comment]
Обновил картинку от Николая. На обсуждение.

![Org_structure](https://user-images.githubusercontent.com/75683852/145071073-8b0751ab-ce77-443e-8865-d4dd8d085f61.png)

И почти аналогичная структура для ИП.

![IP_structure](https://user-images.githubusercontent.com/75683852/145071212-17e5b3ef-817f-423a-b26a-12a54ad5ba1b.png)

**Comment by uestas** [comment]
Прикрепляем вариант переложения части структуры ФРМО на стандартные ресурсы и элементы FHIR (без использования расширений).

Описание структуры маппинга:
•	FHIR столбцы – Ресурс, Элемент FHIR 1 уровня, Элемент FHIR 2 уровня. Рассматривались 3 ресурса ([Organization](http://hl7.org/fhir/organization-definitions.html#Organization), [Location](http://hl7.org/fhir/location-definitions.html#Location), [Device](http://hl7.org/fhir/device-definitions.html#Device)) и один элемент [address](http://hl7.org/fhir/location-definitions.html#Location.address).
•	Поля ФРМО. Наполнение берется из файла "_Структура ФРМО_". Соотносятся с элементами FHIR, которые не противоречат требованиям стандарта. В качестве наполнения допускается перечень полей поскольку в разных случаях возможно разное наполнение.
•	Справочники, которые используются для кодирования полей в соответствии с файлом "_Структура ФРМО_".
Просим Вас ознакомиться и высказать Ваши предложения в формате комментариев в гугл-таблице.

https://docs.google.com/spreadsheets/d/1-WuSdvACCLzjSQQnN3qA1FKmTSMkg6fXlB1NmqOqPsY/edit?usp=sharing

**Reply by Vrodvi** [reply]
Добавил комментарии с предложениями в указанной гугл-таблице, просьба рассмотреть.

> Похоже осуществляется попытка сбросить в "Тип организации" большое количество разнородной по смыслу информации.
> Для многих из перечисленных характеристик в FHIR есть специализированные ресурсы.
> 
> Предлагаю маппировать информацию осмысленно в соответствии с предназначением ресурсов и атрибутов, описанных в FHIR.
> Информацию, которая не укладывается в существующие ресурсы\атрибуты, лучше вынести отдельно на обсуждение, а не пытаться запихнуть в первый попавшийся элемент, допускающий кодирование по произвольному справочнику.
> 
> Некоторые примеры маппирования, нарушающие предназначение данного атрибута FHIR (Organization.type):
> 
> Тип субъекта системы здравоохранения, к которому относится организация
> - для этого больше подходит OrganizationAffiliation между самой организацией и "субъектом СЗ", который тоже оформлен как организация
> 
> Описание вида деятельности, профиль, форма оказания помощи, условия оказания помощи
> - для этого больше подходит ресурс HealthcareService
> 
> Бригада, её профиль, вид, специализация
> - для этого больше подходит рeсурс CareTeam
> 
> Источник финанасирования
> - для этого больше подходит рeсурс Coverage
